install/configure_management_network.md

```diff
@@ -44,6 +44,42 @@ cabinets will have Aggregation switches.
 
 On a typical system, the Mellanox switches are Spine switches and the Dell switches are used for Aggregation, CDU, and Leaf switches.
 
+## Site connections
+
+Currently CANU ***does not*** automatically create site connections (LAGs/uplink interfaces or default routes). 
+
+Hence you need to manually configure the uplinks for site connections as well as default routes. 
+
+Example configuration: 
+
+NOTE: these are very simplistic examples and depending on your install scenario you may need to also configure LAG's etc. 
```

Comment:
Change to:

**NOTE:** The following are very simplistic examples, and depending on the install scenario, additional configuration may be required.